### PR TITLE
feat: add hibernating sessions and project pins from discourse

### PR DESCRIPTION
**Pull request by rokroskar**
Adding some content from discourse

### CODE REVIEW

File: blog/2023-10-11-pause-resume-sessions/index.mdx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+slug: pin-sessions-landing-page
+title: |
+  RenkuLab Update: Pause and Resume Sessions
+authors: [laura]
+tags:
+  - renkulab update
```
Comment (leafty) [inline blog/2023-10-11-pause-resume-sessions/index.mdx:7]:
```suggestion
  - RenkuLab update
```

File: blog/2023-10-11-pause-resume-sessions/index.mdx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  - sessions
```
Comment (leafty) [inline blog/2023-10-11-pause-resume-sessions/index.mdx:8]:
```suggestion
  - Sessions
```

**Review by leafty — COMMENTED** [review]
Some minor things here.

Will check how it renders too.

And, the PR should be renamed to `feat: add X and Y RenkuLab updates`.

**Comment by rokroskar** [comment]
Thanks @leafty - if we think it's a good idea to add these, I can go through and add others from the past ~12-18 months

File: blog/2023-10-11-pause-resume-sessions/index.mdx (hunk 2 above)
Comment (rokroskar) [reply blog/2023-10-11-pause-resume-sessions/index.mdx:8]: it's a tag - should it be capitalized?

Comment (leafty) [reply blog/2023-10-11-pause-resume-sessions/index.mdx:8]: Yes, see how it renders on the tags page: https://blog.renkulab.io/tags.

**Comment by leafty** [comment]
> Thanks @leafty - if we think it's a good idea to add these, I can go through and add others from the past ~12-18 months

Sure, we can have an issue listing the ones we want to add and I can also push some.

**Review by leafty — APPROVED** [review]
LGTM

Note the `h3` title on `pause-resume-session` creates this TOC hierarchy.
Screenshot 2024-02-22 at 10 38 46

**Comment by rokroskar** [comment]
@leafty I updated it to a warning box

image

**Comment by rokroskar** [comment]
> > Thanks @leafty - if we think it's a good idea to add these, I can go through and add others from the past ~12-18 months
> 
> Sure, we can have an issue listing the ones we want to add and I can also push some.

added: https://github.com/SwissDataScienceCenter/renku-blog/issues/32

**Pull request merged**